virtual_ecosystem/core/base_model.py

```diff
@@ -760,3 +760,109 @@ class attribute are present in the :attr:`~virtual_ecosystem.core.data.Data`
             )
             LOGGER.error(error)
             raise error
+
+
+class BaseDisturbance(
+    BaseModel,
+    ABC,
+    model_name="_base_disturbance",
+    model_update_bounds=("1 day", "1 year"),
+    vars_required_for_init=tuple(),
+    vars_populated_by_init=tuple(),
+    vars_required_for_update=tuple(),
+    vars_updated=tuple(),
+    vars_populated_by_first_update=tuple(),
+):
+    """Base class for disturbances in the virtual ecosystem.
+
+    This class extends BaseModel to include disturbance-specific functionality and
+    configuration.
+    """
+
+    disturbed_models: tuple[str, ...] = tuple()
+    """A tuple of model names that this disturbance will affect directly.
+
+    This is used to ensure that the disturbance is applied to the correct models and
+    to validate the disturbance configuration.
+    """
+
+    def __init_subclass__(
+        cls,
+        model_name,
+        model_update_bounds,
+        vars_required_for_init,
+        vars_updated,
+        vars_required_for_update,
+        vars_populated_by_init,
+        vars_populated_by_first_update,
+        disturbed_models: tuple[str, ...] = tuple(),
+    ):
+        """Initialise subclasses deriving from BaseDisturbance."""
+        if vars_populated_by_first_update or vars_populated_by_init:
+            raise ValueError(
+                "Disturbance models cannot define "
+                "vars_populated_by_first_update or vars_populated_by_init."
+            )
+        super().__init_subclass__(
+            model_name,
+            model_update_bounds,
+            vars_required_for_init,
+            vars_updated,
+            vars_required_for_update,
+            tuple(),
+            tuple(),
+        )
+        cls.disturbed_models = disturbed_models
+
+    def __init__(
+        self,
+        data: Data,
+        core_components: CoreComponents,
+        static: bool = False,
+        models: dict[str, BaseModel] | None = None,
+        **kwargs: Any,
+    ):
+        """Initialise the disturbance with shared data and core components.
+
+        This method **must** be called in the ``__init__`` method of all subclasses.
```

Comment:
Are double backticks needed? Would this work?

```suggestion
        This method **must** be called in the `__init__` method of all subclasses.
```

Reply:
This is copy paste from the `testing` model, so I assume it works as it is. If it is not, we need to change it in multiple places.

Reply:
It is RST vs Markdown. At some point I'd like to move the docstrings over to markdown but I don't think there's a good replacement for `sphinx` `autodoc` yet, which can only handle RST docstrings.